### ISSUE TITLE
no player names in notifications

### ISSUE THREAD

**Issue by jakobj** [jakobj/kicker_league#46]
sometimes notifications contain only ids (i.e., numbers) instead of the names of players. it does not happen with my version of the app but apperently with others. did any one else notice this?

**Comment by mschmidt87** [comment]
Yes, from time to time, but I couldn't spot any regularity so far.

**Comment by jakobj** [comment]
it happened again while @alperyeg was debugging the undo bar. does anyone have any idea how this can happen? i looked at the code in the webservice, but the part where ids are translated into names looks proper. @philipp-w could you also have a look again at that part? or do you think the error might be in the app.
